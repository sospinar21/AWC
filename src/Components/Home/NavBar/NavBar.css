.navBar {
  -webkit-box-shadow: 2px 2px 2px #70809094;
  box-shadow: 2px 2px 2px #70809094;
  height: 78vh;
  width: 17vw;
  display: -ms-flexbox;
  -ms-flex-pack: justify;
  justify-content: space-between;
  background: #1b0202;
  z-index: 10; }

.active {
  border-bottom: #cc1292; }

.logo {
  flex: 20 1;
  font-size: 6vh;
  color: white;
  font-family: 'Poiret One', cursive;
  text-shadow: 2px 2px 2px #091d09; }

.menu {
  margin: auto;
  flex: 15 1;
  line-height: 8vh; }

a {
  display: block;
  text-decoration: none;
  color: white;
  font-weight: 600;
  font-size: 2.1vh;
  letter-spacing: 1px;
  margin-left: 2vh; }

.li:hover {
  background-color: palevioletred;
  color: white; }

.logo-box {
  text-align: center;
  padding-top: 2vh;
  margin-bottom: 3vh; }

.li {
  width: 80%;
  display: flex;
  margin: auto; }
