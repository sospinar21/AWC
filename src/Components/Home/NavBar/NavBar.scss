.navBar {
  height: 10vh;
  width: 80vw;
  display: flex;
  justify-content: space-between;
  margin: 0 auto 5vh;
}

.active {
  border-bottom: #cc1292
}

.logo {
  flex: 20;
  font-size: 6vh;
  color: palevioletred;
}

.menu {
  flex: 15;
  line-height: 10vh;
  text-align: center;
}

a {
  text-decoration: none;
    color: black;
    font-weight: 600;
    margin: 5vh;
    padding: 2vh;
    font-size: 2.7vh;
    letter-spacing: 1px;
}

a:hover {
  background-color: palevioletred;
  color: white;
}

.logo {
  font-family: 'Poiret One', cursive;  
}