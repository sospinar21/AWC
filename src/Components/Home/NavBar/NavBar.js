import React, { Component } from 'react';
import { NavLink } from 'react-router-dom';
import { connect } from 'react-redux';
import PropTypes from 'prop-types';
import './NavBar.css';

export class NavBar extends Component {
  
  render () {
    return (
      <div className='navBar'>
        <div className='logo-box'>
          <h1 className='logo'>AWC</h1>
        </div> 
        <div className='menu'>
          <div className = 'li'>
            <i className='material-icons'>account_circle</i>
            <NavLink to='/signin'>SignUp/ LogIn</NavLink>
          </div>
          <div className = 'li'>
            <i className='material-icons'>group</i>
            <NavLink to='/'>Community</NavLink>
          </div>
          <div className = 'li'>
            <i className='material-icons'>whatshot</i> 
            <NavLink to='/entertainment'>Entertainment</NavLink> 
          </div>
        </div>         
      </div>
    );
  }
}

export const mapStateToProps = {};

export const mapDispatchToProps = () => ({});

export default connect(mapStateToProps, mapDispatchToProps)(NavBar);