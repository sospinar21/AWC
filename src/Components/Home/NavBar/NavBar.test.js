import {NavBar,mapDispatchToProps,mapStateToProps } from '../NavBar/NavBar'
import React from 'react';
import ReactDOM from 'react-dom';
import { mount, shallow } from 'enzyme';

describe('navBar', () => {

  let navBar

  beforeEach(() => {

    navBar = mount(<NavBar /> , { disableLifecycleMethods: true })

  })

  it('matches snapshot', () => {
    expect(navBar).toMatchSnapshot();
  })
})