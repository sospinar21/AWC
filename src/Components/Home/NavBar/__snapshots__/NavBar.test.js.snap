// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navBar matches snapshot 1`] = `undefined`;
